### PR TITLE
Fix Architecture Section

### PR DESCRIPTION
**Pull request by ansijain**
Fixes https://github.com/kubevirt/user-guide/issues/348
Signed-off-by: ansijain <ansi.jain@india.nec.com>

### CODE REVIEW

**Comment by mazzystr** [comment]
re https://github.com/kubevirt/kubevirt/pull/5650 ... @fabiand @davidvossel @rthallisey Think we should incorporate this image into the architecture page?

**Comment by ansijain** [comment]
Do you think we should add  [Kubevirt Components Description  ](https://github.com/kubevirt/kubevirt/blob/master/docs/components.md) here?

**Comment by mazzystr** [comment]
> Do you think we should add [Kubevirt Components Description ](https://github.com/kubevirt/kubevirt/blob/master/docs/components.md) here?

@davidvossel I think we should.  How about you?  Something like this should be at the top of the stack so a potential user / architect doesn't have to go looking through the haystack for it.

File: docs/architecture.md [hunk 1]
```diff
@@ -1,5 +1,136 @@
...
+    ~~+---------------------+~~
+      | Orchestration (K8s) |
+      +---------------------+
+      | Scheduling (K8s)    |
+      +---------------------+
+      | Container Runtime   |
+    ~~+---------------------+~~
+      | Operating System    |
+      +---------------------+
+      | (Virtual)           |
```
Comment (rthallisey) [inline docs/architecture.md:20]: kvm

**Review by rthallisey — APPROVED** [review]
lgtm

File: docs/architecture.md (hunk 1 above)
Comment (ansijain) [reply docs/architecture.md]: Updated KVM in stack

**Comment by mazzystr** [comment]
@ansijain Thank you for your contribution

**Pull request merged**